src/Aspire.Dashboard/Components/ResourcesGridColumns/ResourceNameDisplay.razor

```diff
@@ -1,10 +1,24 @@
-﻿@using Aspire.Dashboard.Model
+﻿@namespace Aspire.Dashboard.Components
+
+@using Aspire.Dashboard.Model
+@using Aspire.Dashboard.Otlp.Model
 @using Aspire.Dashboard.Resources
 @inject IStringLocalizer<Columns> Loc
 
-<FluentStack Orientation="Orientation.Horizontal" VerticalAlignment="VerticalAlignment.Center">
-    <span><FluentHighlighter HighlightedText="@FilterText" Text="@FormatName(Resource)" /></span>
-</FluentStack>
+@{
+    var unviewedErrorCount = GetUnviewedErrorCount(Resource);
+}
+
+@if (unviewedErrorCount > 0)
+{
+    <FluentCounterBadge Class="error-counter-badge" Max="9" BackgroundColor="@Color.Error" Color="Color.Fill"  Appearance="Appearance.Lightweight" Count="@unviewedErrorCount" HorizontalPosition="100">
+        <a href="@GetResourceErrorStructuredLogsUrl(Resource)" title="@FormatLogLinkTitle(unviewedErrorCount)"><FluentHighlighter HighlightedText="@FilterText" Text="@FormatName(Resource)" /></a>
```

Comment:
I can imagine that having the resource's name link to its logs would be good for all resources, not just those with errors.

Reply:
@drewnoakes do you think links appearing for all resources would be noisy?

Reply:
It would probably be noisey. It's something to think about in the future. The PR is good as it is.

Reply:
Do we still need to link to the logs from the resource name now that we're reinstated the "view logs" button?